Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM node:alpine
 
-RUN npm install -g yarn
+RUN npm install -g yarn nodemon
```

Comment:
This shouldn't be needed. How are you running the app?

Reply:
make build-docker && make start-docker. I just tried removing it and it still works